pkg/meta/annotations.go

```diff
@@ -339,6 +356,27 @@ func GenNMAResourcesAnnotationName(resourceName corev1.ResourceName) string {
 	return fmt.Sprintf("%s-%s", NMAResourcesPrefixAnnotation, strings.Replace(string(resourceName), ".", "-", 1))
 }
 
+// GenNMAHealthProbeAnnotationName returns the name of the annotation for a specific health probe field.
+func GenNMAHealthProbeAnnotationName(probeName, field string) string {
+	return fmt.Sprintf("vertica.com/nma-%s-probe-%s", probeName, field)
+}
+
+// GetNMAHealthProbeOverride returns the value of a NMA health probe annotation.
+// If the annotation isn't set, or its value doesn't convert to an int, then
+// (0,false) is returned.
+func GetNMAHealthProbeOverride(annotations map[string]string, probeName, field string) (int32, bool) {
```

Comment:
So the user can set any numerical value? There is no threshold for any of these fields?

Reply:
We can probably reject negative values. But there is no upper bound on the values. I'll add a check for negativity.